### PR TITLE
[IMP] Documents documentation to be updated

### PR DESCRIPTION
**Pull request by Donapi**
taskid: 3433587 [https://www.odoo.com/web#id=3433587&cids=1&menu_id=4720&model=project.task&view_type=form](url)

### CODE REVIEW

File: content/applications/finance/documents.rst [hunk 1]
```diff
@@ -69,8 +84,8 @@ organized per category and filters can be used to sort them.
 Documents management
 ====================
 
-When clicking on a specific document, the right panel displays different options. On the top,
-additional options might be available: :guilabel:`Download`, :guilabel:`Share`, :guilabel:`Replace`,
+When selecting a document, the right panel displays different options. On the top, additional
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:87]: When opening a document, the right panel displays different options.

File: content/applications/finance/documents.rst [hunk 2]
```diff
@@ -91,8 +106,88 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
      rights to the document. It is possible to replace the owner of a document. I.e.: an employee
      must be owner of a document to be able to see it in "My Profile".
 
-Finally, different :guilabel:`Actions` are available at the bottom of the right panel, depending on
-the workspace where your document is stored.
+Finally, different :ref:`Actions <documents/workflow-actions>` are available at the bottom of the
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:109]: Different :ref:`Actions <documents/workflow-actions>` are available at the bottom of the

File: content/applications/finance/documents.rst [hunk 3]
```diff
@@ -91,8 +106,88 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
...
+right panel, depending on the workspace where your document is stored.
+
+Split documents
+---------------
+
+Select the document you want to split, and click the :guilabel:`Split` icon. A new page opens and
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:115]: Select the document you want to split, and click the :guilabel:`Split` icon. A new view opens and

File: content/applications/finance/documents.rst [hunk 4]
```diff
@@ -91,8 +106,88 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
...
+
+.. image:: documents/split-pdf.png
+   :align: center
+   :alt: split your documents
+
+.. note::
+
+   To merge documents, select them from your dashboard and click the :guilabel:`Split` icon. Then,
+   click on the scissors between pages to remove the cut, and click :guilabel:`Split`. Your
+   documents are now merged.
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:129]: erase "your documents are now merged"

File: content/applications/finance/documents.rst [hunk 5]
```diff
@@ -91,8 +106,88 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
...
+Select a specific workspace and click on the arrow next to the :guilabel:`Upload` button to access
+additional features:
+
+Request
+~~~~~~~
+
+To request a missing file, click :guilabel:`Request`. Add the :guilabel:`Document name`, select the
+person you need it from in the :guilabel:`Request to` field, fill in the :guilabel:`Due date`, add
+the :guilabel:`Workspace` the document should belong to, and eventually add :guilabel:`Tags` and a
+:guilabel:`Note`. A file reflecting the missing document is created in the workspace.
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:143]: the :guilabel:`Workspace` the document should belong to, and add :guilabel:`Tags` and a
:guilabel:`Note`, if needed.

File: content/applications/finance/documents.rst [hunk 6]
```diff
@@ -91,8 +106,88 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
...
+
+To see all the documents you requested, go to the :guilabel:`Activity` view and to the
+:guilabel:`Requested Document` column.
+
+.. Note::
+
+   To send a **reminder email** to all users you are waiting a document from, click on ⋮, and
+   :guilabel:`Document Request: Reminder`.
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:151]: I don't see how to do this

File: content/applications/finance/documents.rst [hunk 7]
```diff
@@ -91,8 +106,88 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
...
+Add a link
+~~~~~~~~~~
+
+To add a link to your documents dashboard, click on :guilabel:`Add a link`, enter the
+:guilabel:`URL`, and :guilabel:`Name` it. A file reflecting the url is created on your dashboard.
+
+Share
+~~~~~
+
+It is possible to share a document, a subworkspace, or an entire workspace with your contacts.
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:170]: Sharing a document, a subworkspace, or an entire workspace with your contacts

File: content/applications/finance/documents.rst [hunk 8]
```diff
@@ -91,8 +106,88 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
...
+
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:171]: erase space

File: content/applications/finance/documents.rst [hunk 9]
```diff
@@ -91,8 +106,88 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
...
+To share the content of a workspace, select the workspace and click :guilabel:`Share`. A page pops
+up, with a :guilabel:`URL` available. You can :guilabel:`Copy` it to share it with the users you
+want so they can directly access your document workspace and its content. You can :guilabel:`Name`
+the share link and set a validity date for the url by filling the :guilabel:`Valid Until` field.
+Tick the :guilabel:`Include Sub Folders` box if you also want to share subfolders. You can also
+enable users to :guilabel:`Download` or :guilabel:`Download and Upload` documents.
+
+.. note::
+
+   It is possible to only share one or more documents (and not their workspace) by selecting them
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:181]: It is possible to only share one or more documents (without their workspace) by selecting

File: content/applications/finance/documents.rst [hunk 10]
```diff
@@ -91,8 +106,88 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
...
+
+New spreadsheet
+~~~~~~~~~~~~~~~
+
+To create a new spreadsheet, click :guilabel:`New Spreadsheet`. You can select a
+:guilabel:`Blank spreadsheet` or an existing template.
+
+.. _documents/workflow-actions:
 
 Workflow actions
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:192]: Workflow actions help manage documents and overall business operations. These are automated actions that can be created and customized for each workspace. For example, create documents, process bills, sign, organize files, add tags to a file, or move it to another workspace with a single click etc. These workflow actions appear on the right panel when they meet the set criteria.

File: content/applications/finance/documents.rst [hunk 11]
```diff
@@ -44,33 +44,54 @@ Workspaces
...
-   :guilabel:`Workspaces` and :guilabel:`Sub-workspaces` can be created, edited, or deleted by
-   clicking on the gear icon :guilabel:`⚙` on the left menu.
-
-.. image:: documents/sub-workspaces-creation.png
-   :align: center
-   :alt: Create sub-workspaces from the left menu
+- :guilabel:`Name`: add the name of your workspace;
+- :guilabel:`Parent Workspace`: if the workspace belongs to a parent workspace (and is then a
+  sub-workspace), select it from the list;
+- :guilabel:`Company`: if your workspace is only available to the selected company;
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:52]: - :guilabel:`Company`: if your workspace is only available to the selected company.

File: content/applications/finance/documents.rst [hunk 12]
```diff
@@ -44,33 +44,54 @@ Workspaces
...
 
 Tags
-====
+----
 
 Tags are used within workspaces to add a level of differentiation between documents. They are
 organized per category and filters can be used to sort them.
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:58]: Tags are used within workspaces to add a level of differentiation between documents. They are
organized per category, and filters can be used to sort them.

File: content/applications/finance/documents.rst [hunk 13]
```diff
@@ -44,33 +44,54 @@ Workspaces
...
    - Tags can be created and modified by going to :menuselection:`Configuration --> Tags`.
    - Tags can also be created, edited, or deleted, by clicking on the gear icon :guilabel:`⚙`, on
      the left menu.
 
+Access rights
+-------------
+
+From the :guilabel:`Access Rights` tab, you can add :guilabel:`Write Groups` that can see the
+workspace and read, create, and edit its documents. You can also add :guilabel:`Read Groups` to see
+the workspace and read its documents without creation and editing rights.
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:75]: specifiy more about where to find these settings, say to open the cog icon and access rights can be found in the edit section

File: content/applications/finance/documents.rst [hunk 14]
```diff
@@ -91,23 +112,106 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
...
+-------------------
+
+Select a workspace and click on the arrow next to the :guilabel:`Upload` button to access additional
+features:
+
+Request
+~~~~~~~
+
+You can create "missing documents" and organize them as documents, to remind users to download these
+documents.
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:147]: You can create “missing documents” and organize them as documents, to remind users to download them.

File: content/applications/finance/documents.rst [hunk 15]
```diff
@@ -91,23 +112,106 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
...
+
+Select the workspace in which the file should be stored and click :guilabel:`Upload`, then
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:149]: Select the workspace where the file should be stored and click :guilabel:`Upload`, then

File: content/applications/finance/documents.rst [hunk 16]
```diff
@@ -91,23 +112,106 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
...
+the document should belong to, and add :guilabel:`Tags` and a :guilabel:`Note`, if needed. Then,
+click :guilabel:`Request`. A file reflecting the missing document is created in the workspace.
+
+You can see all missing documents by going to the :guilabel:`Activity` view and to the
+:guilabel:`Requested Document` column.
+
+.. Note::
+
+   To send a **reminder email** to all users you are waiting a document from, go to the
+   :guilabel:`Activity` viewclick on ⋮, and :guilabel:`Document Request: Reminder`.
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:161]: I commented before to specify more about where to find activities but I think you didn't add it.

**Review by chiaraprattico — CHANGES_REQUESTED** [review]
- Not clear how to find the domain options even if developer mode.
- Can't find the actions tab? is it actions for documents, workspace?

File: content/applications/finance/documents.rst [hunk 17]
```diff
@@ -44,11 +44,26 @@ Workspaces
...
+From the :guilabel:`Description` tab, you can add descriptive information related to your workspace.
 
 .. note::
-   :guilabel:`Workspaces` and :guilabel:`Sub-workspaces` can be created, edited, or deleted by
-   clicking on the gear icon :guilabel:`⚙` on the left menu.
+   :guilabel:`Workspaces` and :guilabel:`Sub-workspaces` can be created and edited by clicking on
+   the gear icon :guilabel:`⚙` on the left menu.
 
 .. image:: documents/sub-workspaces-creation.png
    :align: center
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/documents.rst]: finally i keep it :)

File: content/applications/finance/documents.rst (hunk 1 above)
Comment (Donapi) [reply content/applications/finance/documents.rst:93]: selecting or opening

File: content/applications/finance/documents.rst [hunk 18]
```diff
@@ -91,8 +106,88 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
...
+
+To add a link to your documents dashboard, click on :guilabel:`Add a link`, enter the
+:guilabel:`URL`, and :guilabel:`Name` it. A file reflecting the url is created on your dashboard.
+
+Share
+~~~~~
+
+It is possible to share a document, a subworkspace, or an entire workspace with your contacts.
+
+To share the content of a workspace, select the workspace and click :guilabel:`Share`. A page pops
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/documents.rst:182]: updated a bit differently

File: content/applications/finance/documents.rst (hunk 16 above)
Comment (Donapi) [reply content/applications/finance/documents.rst:161]: Updated. I think it's clear enough. Let me know if it's still not according to you.

File: content/applications/finance/documents.rst [hunk 19]
```diff
@@ -144,12 +253,9 @@ Advanced condition type: domain
    It is recommended to have some knowledge of Odoo development to properly configure *Domain*
    filters.
 
-To access the *Domain* condition, the :ref:`developer mode <developer-mode>` needs to be activated.
-Once that's done, select the :guilabel:`Domain` condition type, and click on :guilabel:`Add Filter`.
-
-.. image:: documents/activate-domain-condition.png
-   :align: center
-   :alt: Activating the domain condition type in Odoo Documents
+To access the :guilabel:`Domain` condition, the :ref:`developer mode <developer-mode>` needs to be
```
Comment (chiaraprattico) [inline content/applications/finance/documents.rst:256]: mention that we find this option in the action tab

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**